prover-service/src/input_processing/mod.rs

```diff
@@ -24,18 +23,17 @@ use aptos_keyless_common::{
 };
 
 pub fn derive_circuit_input_signals(
-    input: Input,
+    input: VerifiedInput,
     config: &CircuitConfig,
 ) -> Result<(CircuitInputSignals<Padded>, PoseidonHash), anyhow::Error> {
-    // TODO add metrics instead of just printing out elapsed time
     let _span = logging::new_span("DeriveCircuitInputSignals");
 
     let jwt_parts = &input.jwt_parts;
     let epk_blinder_fr = input.epk_blinder_fr;
     let unsigned_jwt_with_padding =
         with_sha_padding_bytes(input.jwt_parts.unsigned_undecoded().as_bytes());
-    let signature = jwt_parts.signature()?;
-    let (temp_pubkey_frs, temp_pubkey_len) = public_inputs_hash::compute_temp_pubkey_frs(&input)?;
+    let (temp_pubkey_frs, temp_pubkey_len) =
```

Comment:
There is still `temp` being used. Please rename all `temp` to `ephemeral`.

Reply:
fixed